### PR TITLE
2021-12-17-bert_base_finnish_uncased_fi

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
+embeddings = BertEmbeddings.pretrained("bert_base_finnish_uncased", "fi") \
+      .setInputCols("sentence", "token") \
+      .setOutputCol("embeddings")
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, embeddings])
+pipeline_model = nlp_pipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
+result = pipeline_model.transform(spark.createDataFrame([['Rakastan NLP: tä']], ["text"]))
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md:49]: Can we find a bigger example from Finnish wikipedia?

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```
+```scala
+...
+val embeddings = BertEmbeddings.pretrained("bert_base_finnish_uncased", "fi")
+      .setInputCols("sentence", "token")
+      .setOutputCol("embeddings")
+val pipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, tokenizer, embeddings))
+val data = Seq("Rakastan NLP: tä").toDF("text")
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md:57]: Can we find a bigger example from Finnish wikipedia?

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+[Download](https://s3.amazonaws.com/community.johnsnowlabs.com/Ahmetemintek/bert_base_finnish_uncased_fi_3.4.0_3.0_1639753349928.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md:42]: Can we add the rest of the components? Document Assembler, SentenceDetector, Tokenizer...

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+embeddings = BertEmbeddings.pretrained("bert_base_finnish_uncased", "fi") \
+      .setInputCols("sentence", "token") \
+      .setOutputCol("embeddings")
+
+nlp_pipeline = Pipeline(stages=[document_assembler, sentence_detector, tokenizer, embeddings])
+pipeline_model = nlp_pipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
+result = pipeline_model.transform(spark.createDataFrame([['Rakastan NLP: tä']], ["text"]))
+```
+```scala
+...
```
Comment (josejuanmartinez) [inline docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md:52]: Can we add the rest of the components? Document Assembler, SentenceDetector, Tokenizer...

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Hi @Ahmetemintek  can you take a look at the requested changes? Thank you!

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md (hunk 1 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md]: I've added new input.

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md (hunk 3 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md]: I've updated code examples.

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md (hunk 4 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md]: I've updated code examples.

File: docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md (hunk 2 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2021-12-17-bert_base_finnish_uncased_fi.md]: I've added new input.

**Review by josejuanmartinez — APPROVED** [review]

**Comment by josejuanmartinez** [comment]
Looks good, thank you!

**Pull request merged**